src/applications/edu-benefits/0994/analytics-events.js

```diff
@@ -0,0 +1,21 @@
+import { SET_SUBMISSION } from '../../../platform/forms-system/src/js/actions';
+import recordEvent from '../../../platform/monitoring/record-event';
+import _ from 'lodash';
+
+const analyticsEvents = [
+  {
+    action: SET_SUBMISSION,
+    event: store => {
+      const state = store.getState();
+      if (_.get(state, 'form.data.privacyAgreementAccepted', {}) === false) {
```

Comment:
I think you can default this to `false` and simplify it a little (because then it should only ever be `true` or `false`):

`if (!_.get(state, 'form.data.privacyAgreementAccepted', false)) {`

Reply:
Agreed. I went ahead and added in the change and removed the explicit privacyAgreeementAccepted value from the prefill data.